sdk/keyvault/Azure.Security.KeyVault.Keys/src/RsaKeyCreateOptions.cs

```diff
@@ -29,6 +29,11 @@ public RsaKeyCreateOptions(string name, bool hsm)
             {
                 KeyType = KeyType.Rsa;
             }
+
+            if(keySize != default)
+            {
+                KeySize = keySize;
+            }
```

Comment:
Perhaps both the parameter keySize and the property KeySize should be nullable so that 0 and not set are distinct cases.